### PR TITLE
[CO-168] Added build job

### PR DESCRIPTION
**Pull request by vseoane**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ENV GEM_HOME /usr/local/bundle
+RUN chmod -R 777 "$GEM_HOME"
+
+USER builder
+ENV HOME /home/builder
+RUN mkdir -p $HOME/app
+ADD --chown=builder:builder . $HOME/app/
+WORKDIR $HOME/app/
+
+RUN gem install bundler -v $BUNDLER_VERSION && bundle install
```
Comment (kenzliang) [inline Dockerfile]: new line

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN apt-get update -qq && \
+    apt-get install -qy curl jq python3 python3-dev python3-pip python3-yaml && \
+    pip3 install awscli && \
+    apt-get clean
+
+RUN useradd -c 'builder of ruby projs' -m -d /home/builder -s /bin/bash builder
+
+ENV GEM_HOME /usr/local/bundle
+RUN chmod -R 777 "$GEM_HOME"
```
Comment (kenzliang) [inline Dockerfile:12]: why 777? and is this only to build the gem?

Comment (vseoane) [reply Dockerfile:12]: following @careyrp's example here 😄

Comment (careyrp) [reply Dockerfile:12]: This a change I pulled from a newer version of the ruby docker image. Without it, only admin can install gems.
https://github.com/docker-library/ruby/blob/master/2.6/alpine3.9/Dockerfile#L129

**Review by kenzliang — APPROVED** [review]

**Pull request merged**